plugins/gitea/tea.go

```diff
@@ -0,0 +1,26 @@
+package gitea
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func giteaCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Gitea CLI",
+		Runs:    []string{"tea"},
+		DocsURL: sdk.URL("https://gitea.com/gitea/tea/src/branch/main/README.md"),
```

Comment:
```suggestion
		DocsURL: sdk.URL("https://docs.gitea.io/en-us/command-line/"),
```

Reply:
As above the command line refers to the server it's self rather than the end user cli tea.